spec/integration/application/agent_spec.rb

```diff
@@ -309,7 +309,7 @@
       ssl_file = tmpfile('systemstore')
       File.write(ssl_file, unknown_ca_cert.to_pem)
 
-      response_proc = -> res {
+      response_proc = -> (req, res) {
```

Comment:
The commit message for this change needs an update. `between` => `before`

Reply:
Updated in https://github.com/puppetlabs/puppet/pull/8054/commits/3aac799bfe540ffb2adad2a3f0b422425f3ff9ef